build.gradle

```diff
@@ -19,10 +19,20 @@ apply plugin: 'com.moowork.node'
 apply plugin: "org.sonarqube"
 apply plugin: 'project-report' // adds dependencyReport and htmlDependencyReport tasks
 
-if (file('localSettings.gradle').exists()) {
-	apply from: 'localSettings.gradle'
+// if localSettings.gradle does not exist lets create one with some sensible defaults
+if (!file('$projectDir/localSettings.gradle').exists()) {
```

Comment:
Why the `$projectDir/` here? The file you create doesn't have it.

Reply:
to be honest because i don't know where gradle will default to. If I do just file('localSettings.gradle') it did also work, I just wanted to be more explicit. On the other hand, there is already a test for file('localSettings.gradle'), so you are right...